pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[project]
+name = "pixi-cuda-environment"
+version = "0.1.0"
+description = "Add a short description here"
+authors = [{name = "Eric Ma", email = "ericmajinglong@gmail.com"}]
+requires-python = ">= 3.11"
+dependencies = ["pandas>=2.2.2,<2.3", "numpy>=2.0.1,<2.1"]
+
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[tool.pixi.project]
+channels = ["conda-forge"]
+platforms = ["osx-arm64", "linux-64"]
+
+[tool.pixi.pypi-dependencies]
+pixi-cuda-environment = { path = ".", editable = true }
+
+[tool.pixi.tasks]
+test = "pytest"
+
+[tool.pixi.dependencies]
+ruff = ">=0.5.5,<0.6"
+pytest = ">=8.3.2,<8.4"
+ipython = ">=8.26.0,<9"
+jax = "*"
+pytest-cov = ">=5.0.0,<6"
+
+# Feature Definitions
+[tool.pixi.feature.cuda-feature]
+platforms = ["linux-64"]
+system-requirements = {cuda = "12"}
+
+[tool.pixi.feature.cuda-feature.dependencies]
+cuda-version = "==12.1"
+cuda-libraries = "*"
+cudnn = "*"
+jaxlib = "* *cuda*"
+__cuda = "*"
+
```

Comment:
Consider adding `linux = "4.4.0"` or similar to the default `system-requirements`, i.e.

```toml
[tool.pixi.system-requirements]
linux = "4.4.0"
```

I recently ran into a situation where the Docker host (GCP Cloud Run to be specific) was running an older `4.4.0` Linux kernel, and without this it refused to install the locked packages.